dissect/target/plugins/os/windows/dpapi/dpapi.py

```diff
@@ -104,6 +117,23 @@ def _load_master_keys_from_path(self, username: str, path: Path) -> dict[str, Ma
                     if not mkf.decrypted:
                         raise Exception("Failed to decrypt System master key")
 
+                passwords = self.keychain()
+                if passwords is not None:
```

Comment:
```suggestion
                if passwords := self.keychain():
```
`passwords` will never be `None`, I assume you mean something like this?